app/src/main.cpp

```diff
@@ -429,15 +361,7 @@ namespace scitt
             "Failed to get host time: {}", ccf::api_result_to_str(result)));
         }
 
-        try
-        {
-          post_entry_common(ctx, host_time, body);
-        }
-        catch (const did::web::AsyncResolutionNeeded& e)
-        {
-          start_asynchronous_registration(ctx, host_time, e.did, body);
-          return;
-        }
+        post_entry_common(ctx, host_time, body);
```

Comment:
nit: I believe we could also remove the `post_entry_continuation`, since it was mainly used to continue the `post_entry` logic after an asynchronous `did:web` resolution.

Similarly, the `post_entry_common` can probably be merged inside this controller since it won't be called anywhere else. But we can still keep that in a separate method if we prefer that.

Reply:
+1 for the merge

Reply:
Agreed, I've merged them and removed `post_entry_continuation`